src/Tizen.Applications.Cion/Tizen.Applications/SecurityInfo.cs

```diff
@@ -0,0 +1,167 @@
+﻿/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Applications
```

Comment:
SecurityInfo가 Cion framework에 국한되어 디자인 된 클래스라면 namespace를 좀 더 구체적으로 조절할 필요가 있어보입니다.
```suggestion
namespace Tizen.Applications.Cion
```